src/dbnode/encoding/ostream_test.go

```diff
@@ -89,7 +89,7 @@ func testWriteBytes(t *testing.T, o OStream) {
 	b, _ := os.Rawbytes()
 	require.Equal(t, rawBytes, b)
 
-	require.Equal(t, 8, os.pos)
+	require.Equal(t, 0, os.pos)
```

Comment:
its a quirk of the implementation of the `ostream` that both pos `0` and pos `8` mean that the last byte is completely full:

```golang
func (os *ostream) hasUnusedBits() bool {
	return os.pos > 0 && os.pos < 8
}
```

That said its better to not change stuff if there is no need and I was perhaps a bit hasty here by changing this test. I'll change my implementation to ensure that it ends up with `pos` set to 8 instead of 0